notebooks/recommender-system.ipynb

```diff
@@ -0,0 +1,843 @@
+{
```

Comment:
<p>"best model" -&gt; "best <em>fit</em> model"?</p><p> </p><p>Here the loss function is a sum of squares error (SSE), maybe you could give the second $J$ a $J^*$ to indicate its a SSE + regularization parameter.</p><p> </p><p>the functions have a number of <code>reshape</code>  calls. </p><p> </p><p>I think </p><p> </p><p><code>J = 0.5 * np.sum((theta.T@X - ratings)**2)</code>  </p><p> </p><p>where <code>theta</code>  are the fitting parameters, <code>X</code>  <span class="ql-cursor">﻿</span></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/cooperrc/uconn_numpy_tutorials/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>